src/main/java/com/example/tily/comment/CommentRequest.java

```diff
@@ -1,22 +1,12 @@
 package com.example.tily.comment;
 
-import lombok.Getter;
-import lombok.Setter;
-
 import javax.validation.constraints.NotBlank;
 
 public class CommentRequest {
 
-    @Getter @Setter
-    public static class CreateCommentDTO{
-
-        @NotBlank(message = "댓글 내용을 입력해주세요.")
-        private String content;
+    public record CreateCommentDTO(@NotBlank(message = "댓글 내용을 입력해주세요.") String content) {
```

Comment:
```suggestion
    public record CreateCommentDTO(
        @NotBlank(message = "댓글 내용을 입력해주세요.");
        String content;
```
이런식으로 줄바꿈을 넣을 수 있다면 가독성을 위해 넣어주세요.